src/firmware/boards/robot_stm32h7/freertos.c

```diff
@@ -320,29 +377,94 @@ void initIoNetworking(void)
         &robot_log_msg, TbotsProto_RobotLog_fields, MAXIMUM_TRANSFER_UNIT_BYTES);
 
     // TODO (#2064) mainboard rev 2.0 doens't have a reset pin for the u-blox chip
-    // so we can't enable the communicator. Uncomment this for mainboard rev 2.1
-    // which should fix this issue.
-    //
-    // GpioPin_t *ublox_reset_pin =
-    //     io_gpio_pin_create(ublox_reset_GPIO_Port, ublox_reset_Pin, ACTIVE_LOW);
-    // io_ublox_odinw262_communicator_init(&huart8, ublox_reset_pin, 5);
+    // so we can't enable the communicator.
```

Comment:
is this comment still relevant?

Reply:
yup, the reset pin is not functional, its just mapped to a placeholder pin rn